query/src/lib/mutation.ts

```diff
@@ -0,0 +1,82 @@
+import { inject, InjectionToken } from '@angular/core';
+import { injectQueryClient } from './query-client';
+import {
+  DefaultError,
+  MutationObserver,
+  type MutationObserverOptions,
+  type MutationObserverResult,
+  notifyManager,
+} from '@tanstack/query-core';
+import { firstValueFrom, isObservable, Observable } from 'rxjs';
+import { toSignal } from '@angular/core/rxjs-interop';
+
+type CreateMutationOptions<
+  TData = unknown,
+  TError = DefaultError,
+  TVariables = void,
+  TContext = unknown
+> = Omit<
+  MutationObserverOptions<TData, TError, TVariables, TContext>,
+  'mutationFn'
+> & {
+  mutationFn: (variables: TVariables) => Promise<TData> | Observable<TData>;
+};
+
+class Mutation {
```

Comment:
Can you please add a mutation page with an example to the playground?

Reply:
Sure, no problem!